milestone2.ts

```diff
@@ -150,11 +173,14 @@ let updateVolunteer: UpdateVolunteer; // Code here
 type GetVolunteerInfo = (vol: Volunteer) => String;
 let getVolunteerInfo: GetVolunteerInfo = (vol) => {
   let name, age, city; // Code here
+  name = vol.name;
+  age = vol.age;
+  city = vol.city;
```

Comment:
Should use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring)